docker-compose.yml

```
@@ -16,22 +17,25 @@ services:
       retries: 5
   sandbox:
     image: ghcr.io/khoj-ai/terrarium:latest
-    ports:
-      - "8080:8080"
+    restart: always
+    networks:
+      - default
   search:
     image: docker.io/searxng/searxng:latest
-    ports:
-      - "42113:8080"
+    restart: always
+    networks:
+      - default
     volumes:
       - khoj_search:/etc/searxng
     environment:
-      - SEARXNG_BASE_URL=http://localhost:42113/
+      - SEARXNG_BASE_URL=http://0.0.0.0:8080/
```

Can we change this to `localhost`? Otherwise it can conflict with other services running at `8080` on the host machine.

```suggestion
      - SEARXNG_BASE_URL=http://localhost:8080/
```

---

@sabaimran I fixed it. :)